src/apps/Permissions/ManageRolePanel.js

```diff
@@ -104,8 +104,8 @@ class ManageRolePanel extends React.PureComponent {
     return isEmptyAddress(manager.address)
       ? CREATE_PERMISSION
       : isBurnEntity(manager.address)
-        ? VIEW_PERMISSION
-        : updateAction
+      ? VIEW_PERMISSION
```

Comment:
Hmmm, personally I think the extra tab helps to understand the second level of ternary...